### DISCUSSION TITLE
RSL install fails on Apple silicon

### DISCUSSION THREAD

**Discussion by mvanlierwalq** [ARM-DOE/pyart#1244] [General]
RSL code is a bit out-of-date and fails compiling with the modern gcc that ships with Apple silicon computers. I get the following sort of error message:

wsr88d_to_radar.c:456:11: error: implicit declaration of function 'wsr88d_remove_sails_sweep' is invalid in C99 [-Werror,-Wimplicit-function-declaration] wsr88d_remove_sails_sweep(radar); ^ 1 error generated. make[1]: *** [wsr88d_to_radar.lo] Error 1

Fix (thanks Max Grover!) appears to be to set env as follows: export CFLAGS="--std=c90 -Wno-implicit-function-declaration"
Make sure you do make clean before trying again after a fail.

**Comment by mgrover1** [comment]
CFLAGS to pass in to get TRMM_RSL to run on the new Apple silicon chips:

```bash
export CFLAGS="--std=c90 -Wno-implicit-function-declaration"
```
